### DISCUSSION TITLE
Ability to always make form actions sticky

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#6232] [Ideas]
This is related to https://github.com/filamentphp/filament/pull/5810. 
It might be better to show an user that they need to save something if the form buttons would always show like the sticky form action. I imagine something like a config option.
What do you think about it?

**Comment by danharrin** [comment]
I don't think I know what you mean

**Reply by Wulfheart** [reply]
So #5810 adds the ability to make form actions sticky if the buttons are out of the viewport. This is an option I personally like but it only applies if the form actions don't fit on the page. 
My proposal: To make it clearer for users that they need to click save (or cancel) on a change I think it could be beneficial that an option exists that displays the form action buttons as sticky. This could make it more clear to the user that they edited something and need to perform an action (like save or cancel). 

Is it now a little bit clearer? Sorry that I didn't elaborate it further previously.

**Reply by danharrin** [reply]
If you want to make a PR to add that go ahead, but I'm not sure if it would look out of place or not. So no guarantee it would be merged but its maybe worth a try if you want the feature.

Alternatively you could probably do this with custom CSS.
